common/lib/models/poll.dart

```diff
@@ -36,32 +36,39 @@ class Poll {
   bool get isActive => end.isAfter(DateTime.now());
 
   String get endFormatted => prettyDuration(
-        end.difference(DateTime.now()),
-        tersity: DurationTersity.minute,
-        spacer: '',
-        conjunction: ', and ',
-        abbreviated: true,
-      );
+    end.difference(DateTime.now()),
+    tersity: DurationTersity.minute,
+    spacer: '',
+    conjunction: ', and ',
+    abbreviated: true,
+  );
 
+  Map<String,dynamic> get details =>
```

Comment:
What is this used for?

Reply:
Probably the best way to do this would be to make a parent class for both poll and comment to have it contain common functions between them. 

Currently, details is a function which treats both comments and polls in the same way so that the app can display text on either the poll or comment page without needing to write a separate dart file.